kafka-connect-s3/docs/s3_connector.rst

```diff
@@ -251,6 +253,83 @@ Configuration
 This section gives example configurations that cover common scenarios. For detailed description of all the
 available configuration options of the S3 connector go to :ref:`Configuration Options<s3_configuration_options>`
 
+Configurable credential provider
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Some use cases require more fine-grained credentials configuration for AWS so that each connector
+could have its own credentials.
+
+To use configurable credential provider, ``s3.credentials.provider.class`` should be a class which
```

Comment:
nit: "... should be a class ~~which~~*that*..."

Reply:
👍 fixed